### ISSUE TITLE
Support piano staves in all backends (tracking)

### ISSUE THREAD

**Issue by hanshoglund** [music-suite/music-score#323]
TODO list deps

Depends on #109 (amongst others).

**Comment by hanshoglund** [comment]
#324 related. Close for now.

**Comment by hanshoglund** [comment]
This repo has been merged into: https://github.com/music-suite/music-suite

Closing :)
